### DISCUSSION TITLE
Show multiple errors

### DISCUSSION THREAD

**Discussion by roylisto** [cue-lang/cue#1538] [Q&A] [answered]
Hi, I have defined a YAML that contains multiple fields that have errors values, but when I run the cue vet for example
`cue vet test.yaml test.cue`
it only shows the 1st field error, then if I fix the 1st field error, and run the command again, it shows the 2nd field error, so I need to run multiple times to fix the YAML files.
is there any way to run and show all the errors at 1 time only? 
I have add -E flag, but didn't work as expected
I am still using v.4.0

Thank you

**Comment by rogpeppe** [comment]
Could you share a complete example that demonstrates your issue, please?

**Reply by roylisto** [reply]
test.cue
```
hello: "world"
foo: "bar"
```

test.yaml
```
hello: test
foo: test
```

try run this `cue vet test.yaml test.cue`

the error only mentions about `hello` 
```
hello: conflicting values "world" and "test":
    ./test.cue:1:8
    ./test.yaml:1:9
```

I am expecting it's also tell the `foo` value is wrong
So I need to fix the `hello` value first at yaml file to know that `foo` is also has a wrong value

**Comment by slewiskelly** [comment, marked as answer]
The desired behavior is the default in >= `v.0.4.2` (https://github.com/cue-lang/cue/commit/04812bfb515de3300ac21ca181542f8ea02c9ea1):

```console
$ cue version
cue version v0.4.2 darwin/amd64

$ cue eval test.cue test.yaml
foo: conflicting values "bar" and "test":
    ./test.cue:2:8
    ./test.yaml:2:7
hello: conflicting values "world" and "test":
    ./test.cue:1:8
    ./test.yaml:1:9
```

**Reply by roylisto** [reply]
wow thanks, I will try to update my cue version

**Reply by roylisto** [reply]
yeah it works on the cue binary, but if I use the Validate() function in the go library, it still not working for me hmm

**Reply by eonpatapon** [reply]
did you use `errors.Details` to print errors ?

(package cuelang.org/go/cue/errors)

```go
err := v.Validate()
if err != nil {
   fmt.Println(errors.Details(err, &errors.Config{Cwd: dir}))
}
```

**Reply by roylisto** [reply]
I try to use errors.Details, it still throws the 1st error and does not return the 2nd error

this is the output if I use errors.Details
```
metadata.annotations.verified: 2 errors in empty disjunction:
metadata.annotations.verified: conflicting values "false" and "true2":
    /service_component.cue:27:54
    /service_component.cue:59:1
    yaml.Validate:11:35
metadata.annotations.verified: conflicting values "true" and "true2":
    /service_component.cue:27:45
    /service_component.cue:59:1
    yaml.Validate:11:35
```
